wls-admin-service/src/main/java/de/muenchen/oss/wahllokalsystem/adminservice/service/wahltermindaten/WahltermindatenValidator.java

```diff
@@ -0,0 +1,20 @@
+package de.muenchen.oss.wahllokalsystem.adminservice.service.wahltermindaten;
+
+import de.muenchen.oss.wahllokalsystem.adminservice.exception.ExceptionConstants;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.util.ExceptionFactory;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class WahltermindatenValidator {
+
+    private final ExceptionFactory exceptionFactory;
+
+    public void validWahltagIDParamOrThrow(final String wahltagID) {
+        if (wahltagID == null || StringUtils.isEmpty(wahltagID) || StringUtils.isBlank(wahltagID)) {
```

Comment:
```suggestion
        if (StringUtils.isBlank(wahltagID)) {
```
Würde isBlank alleine nicht schon alles abdecken und somit ausreichen?